frontend/webpack.config.js

```diff
@@ -42,4 +39,9 @@ module.exports = {
     modules: ['node_modules'],
     extensions: ['.js', '.jsx']
   },
+  devServer: {
+    devMiddleware: {
+      writeToDisk: true,
+    }
+  }
```

Comment:
Is this needed if you're using `webpack --watch`? You might be able to remove `webpack-dev-server` from `package.json` as well.

https://v4.webpack.js.org/guides/development/#using-watch-mode

Reply:
Seems like it might not be needed.